src/main/scala/ai/privado/utility/UnresolvedReportUtility.scala

```diff
@@ -62,10 +62,10 @@ object UnresolvedReportUtility {
             )
           })
 
-        if (filename.equals(Constants.JAVA_STATS)) {
-          unresolvedNamespaces = cpg.call.methodFullName(java_unresolved_namespace).l.length
+        if (statfilepath.endsWith(Constants.JAVA_STATS)) {
```

Comment:
Instead of checking on the Filepath take `language` as a input to the function and use that to differentiate